src/game.js

```diff
@@ -0,0 +1,24 @@
+import { Board } from '../src/board';
+import { Player } from '../src/player';
+
+class Game {
+  constructor() {
```

Comment:
This could be a good place to use [dependency injection](https://martinfowler.com/articles/injection.html)